app/grandchallenge/retina_importers/views.py

```diff
@@ -219,6 +220,23 @@ def set_element_data_file_header(self, mhd_file, raw_file_name):
             sys.getsizeof(new_file),
         )
 
+    def save_mhd_and_raw_files(self, request, img):
+        # Save MHD and ZRAW files to Image.files model
+        uuid_name_str = str(uuid.uuid4())
```

Comment:
Everything so far is called `out.{mhd|zraw}` - is it better to stick to this convention? We will update it later anyway to `images/<image_pk>/<imagefile_pk>.<imagefile_type>`.

Reply:
Ok, done in a7002e0bb5e4347d2d6ef223b6d168f80db8fe54